docs/autonomy.md

```diff
@@ -14,180 +14,119 @@ Commands:
-│   ├── benchmarks
-│   ├── logs
-│   ├── tm_state
-│   └── venvs
-└── docker-compose.yaml
-```
-
-To run this deployment go to the `abci_build` and run `docker-compose up`.
-
-
-2. Deployment images
-
-```bash
-Usage: autonomy build-image [OPTIONS] [PUBLIC_ID_OR_HASH]
-
-  Build image using skaffold.
-
-Options:
-  --service-dir PATH  Path to service dir.
-  --pull              Pull latest dependencies.
-  --help              Show this message and exit.
-```
-
-```bash
-# To build an image, navigate to the service dir and run
-$ autonomy build-image
-```
-
-This will create and image with label `valory/open-autonomy-open-aea:oracle-0.1.0`. Images generated by this command will follow `valory/open-autonomy-open-aea:app_name-version` format for labels.
-
-If you just want to build image for a specific agent without a fetching service run
-
-```bash
-$ autonomy build-image PUBLIC_ID
-```
+### Build tools
+
+1. Service deployment
+
+    ```bash
+    Usage: autonomy deploy build [OPTIONS] [KEYS_FILE]
+
+      Build deployment setup for n agents.
+
+    Options:
+      --o PATH                        Path to output dir.
+      --n INTEGER                     Number of agents.
+      --docker                        Use docker as a backend.
+      --kubernetes                    Use kubernetes as a backend.
+      --dev                           Create development environment.
+      --force                         Remove existing build and overwrite with new
+                                      one.
+      --log-level [INFO|DEBUG|WARNING|ERROR|CRITICAL]
+                                      Logging level for runtime.
+      --packages-dir PATH             Path to packages dir (Use with dev mode)
+      --open-aea-dir PATH             Path to open-aea repo (Use with dev mode)
+      --open-autonomy-dir PATH        Path to open-autonomy repo (Use with dev
+                                      mode)
+      --aev                           Apply environment variable when loading
+                                      service config.
+      --use-hardhat                   Include a hardhat node in the deployment
+                                      setup.
+      --use-acn                       Include an ACN node in the deployment setup.
+      --image-version TEXT            Define runtime image version.
+      --remote                        To use a remote registry.
+      --local                         To use a local registry.
+      -p                              Ask for password interactively
+      --password PASSWORD             Set password for key encryption/decryption
+      --help                          Show this message and exit.
+    ```
+
+    To execute this command you need to be located within a service folder. That is, a folder containing the service definition file (`service.yaml`) and a file containing keys with funds for the chain you want to use. The deployment will be created in the subfolder `./abci_build`.
+
```

Comment:
Please add a reference to a tutorial where we use this command

Reply:
It's in line 129 of the file. Is that ok?